src/bots/pauser.py

```diff
@@ -56,6 +58,15 @@ def __init__(self, w3: Web3):
                 )
             )
 
+        if TransportType.DATA_BUS in variables.MESSAGE_TRANSPORTS:
+            transports.append(
+                DataBusProvider(
```

Comment:
provide `DATA_BUS_ADDRESS` to constructor here

Reply:
Done